tools/cli/build.gradle

```diff
@@ -38,9 +38,12 @@ task copyCLIShortcut(type: Copy, dependsOn: [distBinary, dumpOSInfo]) {
     String go_osarch     = mapOsArchToGoArch(getOsArch())
     String from_path_wsk = "${projectDir}/../../bin/${go_osname}/${go_osarch}/wsk"
     String to_path_dir   = "${projectDir}/../../bin"
-
-    from from_path_wsk
-    into to_path_dir
+    if (file(from_path_wsk).exists()) {
+      exec {
```

Comment:
@bpoole16, Remove the changes to the build.gradle and I say we can merge this one after Travis passes. Will resolve your build issues in a different PR.